src/tilemap.frag

```diff
@@ -12,5 +13,5 @@ void main(void)
 
    vec4 color;
    %forloop%
-   gl_FragColor = color;
-}
\ No newline at end of file
+   gl_FragColor = (color * vAlpha) + (gl_FragColor * (1.0 - vAlpha));
```

Comment:
I don't think you're supposed to read from `gl_FragColor` if haven't set it. Can you test this:

```suggestion
   gl_FragColor = color * vAlpha;
```